docs/references/architecture/adr-113-mempool-lanes.md

```diff
@@ -96,27 +109,32 @@ Before stating this property, we need to make some definitions.
 characteristics as defined by the application.
 
 A transaction may only have one class. If it is not assigned any specific class, it will be assigned
-a *default class*, which is a special class always present in any set of classes. Because no
-transaction can belong to two or more classes, transaction classes form disjoint sets, that is, the
-intersection between classes is empty. Also, all transactions in the mempool are the union of the
-transactions in all classes.
+a *default class*, which is a special class always present in any set of classes. This is analogous
+to the _native VLAN_ for untagged traffic in an 802.1Q network. Because no transaction can belong to
+two or more classes, transaction classes form disjoint sets, that is, the intersection between any two
+classes is empty. Also, all transactions in the mempool are the union of the transactions in all
+classes.
 
 :memo: _Definition_: Each class has a *priority* and two classes cannot have the same priority.
 Therefore all classes can be ordered by priority.
 
+When a transaction is received for the first time and validated via `CheckTx`, the application MUST
```

Comment:
But, we say somewhere that, if the add doesn't return a class, then the TX belongs to the default class?

Reply:
True. I fixed it.